com.microsoft.mrtk.graphicstools.unity/Samples~/MaterialGallery/Materials/ShaderBallEmissiveMap.mat

```diff
@@ -0,0 +1,225 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: ShaderBallEmissiveMap
+  m_Shader: {fileID: 4800000, guid: c331f6c43a2ef0945864cb668f2653c9, type: 3}
+  m_ShaderKeywords: _DIRECTIONAL_LIGHT _EMISSION _HOVER_LIGHT _METALLIC_TEXTURE_ALBEDO_CHANNEL_A
+    _NORMAL_MAP _REFLECTIONS _SPECULAR_HIGHLIGHTS _USE_WORLD_SCALE
```

Comment:
nitpick: the material has normal mapping enabled but no normal map texture. Maybe we should disable normal mapping? And can you remove the missing script component?

![image](https://user-images.githubusercontent.com/13305729/178032845-784e868f-8c84-4b0f-857a-b1f7f70373d7.png)

Reply:
Has been taken care of.

Reply:
Awesome, please feel free to squash and merge when you are ready!